rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -116,3 +116,11 @@ horizon_custom_uploads:
   favicon:
     src: "{{ rackspace_static_files_folder }}/favicon.ico"
     dest: img/favicon.ico
+
```

Comment:
We were previously seding it in the variables file from OSA, however we are not copying that over as it is entirely commented out and we can't copy OSA's variables and RPC-O's OSA variables to the same location, so I copied this portion.  The other portions were easier to change by changing the sed's to echo, but this had extra logic and I wanted to keep the comments around it in the variables file.